### DISCUSSION TITLE
Set refresh rate in iOS calendar app

### DISCUSSION THREAD

**Discussion by AlexHupe** [spatie/icalendar-generator#124] [Q&A]
Hi, 

in my source code I have set the refresh interval to 60 minutes. But I tried 15 minutes, too. 

`BEGIN:VCALENDAR
VERSION:2.0
PRODID:spatie/icalendar-generator
NAME:Borussia Dingelstädt e.V.
X-WR-CALNAME:Borussia Dingelstädt e.V.
DESCRIPTION:Kalender von Alex Hupe
X-WR-CALDESC:Kalender von Alex Hupe
REFRESH-INTERVAL;VALUE=DURATION:PT60M`

Despite this, is the refresh period in the dialog box from iOS "weekly".

Can you help me?

Best, Alex

**Comment by rubenvanassche** [comment]
As far as I'm aware IOS completely ignores the property, sadly enough.
